src/Apps/CollectorProfile/Routes/Saves2/Components/SelectListsForArtworkModal/SelectListsForArtworkModal.tsx

```diff
@@ -14,75 +14,56 @@ import { useUpdateCollectionsForArtwork } from "Apps/CollectorProfile/Routes/Sav
 import createLogger from "Utils/logger"
 import { useTranslation } from "react-i18next"
 import { SelectListsForArtworkContent } from "Apps/CollectorProfile/Routes/Saves2/Components/SelectListsForArtworkModal/SelectListsForArtworkContent"
+import {
+  ListKey,
+  useManageArtworkForSavesContext,
+} from "Components/Artwork/ManageArtworkForSaves"
 
 const logger = createLogger("SelectListsForArtworkModal")
 
-interface SelectListsForArtworkModalQueryRenderProps {
-  artworkID: string
-  onClose: () => void
-  onSave: (collectionIds: string[]) => void
-}
-
 export interface SelectListsForArtworkModalProps {
   me: SelectListsForArtworkModal_me$data | null
   artwork: SelectListsForArtworkModal_artwork$data | null
-  onClose: () => void
-  onSave: (collectionIds: string[]) => void
 }
 
 export const SelectListsForArtworkModal: React.FC<SelectListsForArtworkModalProps> = ({
   me,
   artwork,
-  onClose,
-  onSave,
 }) => {
   const { t } = useTranslation()
+  const { state, dispatch, onSave } = useManageArtworkForSavesContext()
```

Comment:
ty for removing of all of the props passing! much less brittle

Reply:
you're welcome 😅